esp/scm/ws_elk.ecm

```diff
@@ -0,0 +1,27 @@
+ESPRequest GetConfigDetailsRequest
+{
+};
+
+ESPResponse GetConfigDetailsResponse
+{
+    boolean IntegrateKibana;
+    string KibanaAddress;
+    string KibanaPort;
+    string KibanaEntryPointURI;
+    boolean ReportElasticSearchHealth;
+    string ElasticSearchAdresses;
+    string ElasticSearchPort;
+    boolean ReportLogStashHealth;
+    string LogStashAddress;
+    string LogStashPort;
+ };
+
+ESPservice [auth_feature("ESDLConfigAccess:ACCESS"), version("1.0"), default_client_version("1.0"), exceptions_inline("xslt/exceptions.xslt")] ws_elk
+{
+    ESPmethod getConfigDetails(GetConfigDetailsRequest, GetConfigDetailsResponse);
```

Comment:
I prefer that the method name starts with Capital ('G') as other ESP services.

Reply:
Indeed! Not sure how that slipped through